squad-list-test

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import logging
+import requests
+import sys
+from squad_client.core.api import SquadApi
+from squad_client.core.models import ALL, Squad, SquadObjectJSONEncoder, TestRun
+from squad_client.utils import first, getid
+
+SquadApi.configure(cache=3600, url="https://qa-reports.linaro.org/")
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+def arg_parser():
+    parser = argparse.ArgumentParser(description="List data about a test")
+
+    parser.add_argument(
+        "--group",
+        required=True,
+        help="squad group",
+    )
+
+    parser.add_argument(
+        "--project",
+        required=True,
+        help="squad project",
+    )
+
+    parser.add_argument(
+        "--build",
+        required=True,
+        help="squad build",
+    )
+
+    parser.add_argument(
+        "--environment",
+        required=True,
+        help="squad environment",
+    )
+
+    parser.add_argument(
+        "--suite",
+        required=True,
+        help="squad suite",
+    )
+
+    parser.add_argument(
+        "--test",
+        required=True,
+        help="squad test",
+    )
+
+    return parser
+
+def run():
+    args = arg_parser().parse_args()
+
+    group = Squad().group(args.group)
+    if group is None:
+        logger.error(f"Get group failed. Group not found: '{args.group}'.")
+        return -1
+
+    project = group.project(args.project)
+    if project is None:
+        logger.error(f"Get project failed. Project not found: '{args.project}'.")
+        return -1
+
+    build = project.build(args.build)
+    if build is None:
+        logger.error(f"Get build failed. Build not found: '{args.build}'.")
+        return -1
+
+    environment = project.environment(args.environment)
+    if environment is None:
+        logger.error(f"Get environment failed. Environment not found: '{args.environment}'.")
+        return -1
+
+    suite = project.suite(args.suite)
+    if suite is None:
+        logger.error(f"Get suite failed. Suite not found: '{args.suite}'.")
+        return -1
+
+    test = first(Squad().tests(
```

Comment:
Here you could use `build.tests(` and avoid `build=build.id`, but it works the same way.